src/DomainGuest.sol

```diff
@@ -205,36 +202,17 @@ abstract contract DomainGuest {
 
     // --- MCD Support ---
 
-    /// @notice Record changes in line and grain and update dss global debt ceiling if necessary
+    /// @notice Record changes in Line
     /// @param _lid Local ordering id
     /// @param wad The new debt ceiling [WAD]
     function _lift(uint256 _lid, uint256 wad) internal ordered(_lid) {
```

Comment:
May want to consider change `wad` to a `rad`.

Reply:
Did the change. Let me know what you think, my only doubt if `DomainHost.lift` should still log the `wad` param or the `rad` value being passed to the `Guest`(what I did).

Reply:
Ya it feels a bit excessive to log both the wad and rad, and I can see arguments for having it as either a wad or a rad. The rad precisely captures the wad argument in the DomainHost though, so I think it's fine to log the rad. I suspect the comparison of the event will mostly be to match it with DomainHost and DomainGuest so best to have it be the same precision.